Dockerfile.prefect

```diff
@@ -0,0 +1,14 @@
+FROM python:3.10-slim
+
+WORKDIR /deployments
+
+COPY . .
+
+RUN apt-get update \
+    && apt-get -y install libpq-dev gcc \
+    && pip install psycopg2
```

Comment:
Why is this required?

Reply:
I don't use the Dockerfile anymore so it's not needed  anymore. These are needed if you're using sqlalchemy inside a Docker container.